dpdata/plugins/lammps.py

```
@@ -108,7 +108,9 @@ def from_system(
         register_charge(data)
         return data
 
-    def to_system(self, data, file_name: FileType, frame_idx=0, **kwargs):
+    def to_system(
+        self, data, file_name: FileType, frame_idx=0, output_spins=False, **kwargs
```

Ruff (0.12.2)

Unused method argument: `kwargs`

(ARG002)